src/structure.json

```diff
@@ -1,7 +1,15 @@
 {
+  "topnav" : 
```

Comment:
align indentation of topnav  and menu with other content

Reply:
This is a little confusing, what exactly is not correct there  ?

Reply:
changed